### DISCUSSION TITLE
Welcome to python-keylogger Discussions!

### DISCUSSION THREAD

**Discussion by secureyourself7** [secureyourself7/python-keylogger#2] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by chukxoka4** [comment]
Hi, I'm @chukxoka4, a developer myself. However I am looking for a Python Developer from Romania to fill a Python Full Stack and Python/Scala Backend Developer role. I was hoping I would be able to connect with developers interested in this type of opportunity.
